presto-main/src/main/java/com/facebook/presto/cost/CachingCostProvider.java

```diff
@@ -35,7 +77,7 @@
     private final Session session;
     private final TypeProvider types;
 
-    private final Map<PlanNode, PlanNodeCostEstimate> cache = new IdentityHashMap<>();
+    private final Map<PlanNode, PlanCostEstimate> cache = new IdentityHashMap<>();
```

Comment:
It seems that what is called `PlanCostEstimate` must be called `PlanNodeCostEstimate`. And what is currently called `PlanNodeCostEstimate` must be called `PlanCostEstimate`